packages/components/src/components/hds/accordion/index.hbs

```diff
@@ -4,5 +4,5 @@
 }}
 
 <div class="hds-accordion" ...attributes>
-  {{yield (hash Item=(component "hds/accordion/item"))}}
+  {{yield (hash Item=(component "hds/accordion/item" forceState=@forceState))}}
```

Comment:
playing advocate's devil: should this new argument called something like `forceIsOpen` (or `forceOpen`) and be a boolean? 🤔

Reply:
another consideration: I was imagining that the forced state would be at the item level, not at the parent accordion level (the `isOpen` is at item level). why is it necessary at this level? one can achieve the same "collapse all/expand all" by passing the `@forceState` argument to all the items, plus (in that case) it's possible to control a single item independently from the others (while with the current implementation it's not possible)

Reply:
my understanding is that our consumers want a way to manage the states of all accordion items (not individual items), hence the approach of setting it at the root and cascading it to items.

Reply:
updated with the suggested adjustments, adding `onClickToggle` to `DisclosurePrimitive` and forwarding it to `Accordion::Item`

Reply:
OK, I understand.